Update files-to-onedrive.md (#10453)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/setup/files-to-onedrive.md

In addition to working on your OneDrive files locally, you can access them from your web browser:

1. In your web browser, sign in to office.com with your work email.
2. Choose  **OneDrive**. All of your files are listed, and you can work on them in the browser from any computer.

> [!NOTE]  

> > When multiple users use shared computer and sync files via onedrive locally, the storage could get consumed very quickly.

Reviewer: ```suggestion
> When multiple users use shared computer and sync files via onedrive locally, the storage could get consumed very quickly.
```
```suggestion
> When multiple users use a shared computer and sync files via OneDrive locally, the available storage might  be consumed very quickly.
```

Author: It says it is outdated

Author: @JohanFreelancer9

> > When multiple users use shared computer and sync files via onedrive locally, the storage could get consumed very quickly.

Reviewer: ```suggestion
> When multiple users use a shared computer and sync files via OneDrive locally, the available storage might be consumed very quickly.
```
